src/config.ts

```diff
@@ -1,22 +1,29 @@
 const kafka = {
   consumer: {
-    autoCommit : true,
-    fetchMaxBytes : 1048576,
-    fetchMaxWaitMs : 1000,
-    group : "subscription-engine",
-    id : "consumer-1",
+    autoCommit: true,
+    fetchMaxBytes: 1048576,
+    fetchMaxWaitMs: 1000,
+    group: "subscription-engine",
+    id: "consumer-1",
   },
-  kafkaAddress: process.env.DATABROKER_KAFKA_ADDRESS || "kafka",
-  kafkaPort: process.env.DATABROKER_KAFKA_PORT || 9092,
-  zookeeper: process.env.DATABROKER_ZOOKEEPER_HOST || "zookeeper:2181",
+  kafkaAddress: process.env.KAFKA_ADDRESS || "kafka",
+  kafkaPort: process.env.KAFKA_PORT || "9092",
+  zookeeperAddress: process.env.ZOOKEEPER_ADDRESS || "zookeeper",
```

Comment:
For now we are still using zookeeper because of TopicManager's producer. We'll remove that in the future.